brianlong/www.validators.app
# 183580524 remove cluster stats page

#### What's this PR do?
- remove cluster stats page and report

#### How should this be manually tested?
- run tests
- click through page to see if there's no link to cluster stats

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183580524)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
